articles/operator-service-manager/best-practices-onboard-deploy.md

```diff
@@ -8,164 +8,173 @@ ms.topic: best-practice
+- When unsure, a good starting point is to expose the parameter and have a reasonable default specified in the CGS. Below is the sample CGS and corresponding CGV payloads.
+- A single User Assigned Managed Identity should be used in all the Network Function ARM templates and should be exposed via CGS.
 
-In this scenario, we recommend that a single global CGS is used to expose the common NFs’ and third party components’ configurations. NF-specific CGS can be defined as needed.
+CGS payload:
+<pre>
+{ 
+  "type": "object", 
+  "properties": { 
+    "abc": { 
+    "type": "integer", 
+    <b>"default": 30</b>
+    }, 
+    "xyz": { 
+    "type": "integer", 
+    <b>"default": 40</b>
+    },
+    "qwe": {
+    "type": "integer" //doesn't have defaults
+    }
+  }
+  "required": "qwe"
+}
+</pre>
 
-### Choose exposed parameters
+Corresponding CGV payload passed by the operator:
 
-General recommendations when it comes to exposing parameters via CGS: 
+<pre>
+{
+"qwe": 20
+}
+</pre>
 
-- CGS should only have parameters that are used by NFs (day 0/N configuration) or shared components.
+Resulting CGV payload generated by AOSM:
+<pre>
+{
+"abc": 30,
+"xyz": 40,
+"qwe": 20
+}
+</pre>
 
-- Parameters that are rarely configured should have default values defined.
+## Configuration Group Values considerations
 
-- When multiple CGSs are used, we recommend there's little to no overlap between the parameters. If overlap is required, make sure the parameters names are clearly distinguishable between the CGSs.
+Before submitting the CGV resource creation, you can validate that the schema and values of the underlying YAML or JSON file match what the corresponding CGS expects. To accomplish that, one option is to use the YAML extension for Visual Studio Code.
 
-- What can be defined via API (AKS, Azure Operator Nexus, Azure Operator Service Manager (AOSM)) should be considered for CGS. As opposed to, defining those configuration values via CloudInit files.
+## CLI considerations
 
-- A single User Assigned Managed Identity should be used in all the Network Function ARM templates and should be exposed via CGS.
+The AOSM CLI extension assists with the publishing of NFDs and NSDs. Use this tool as the starting point for creating new NFD and NSD. Consider using the CLI to create the initial files and then edit them to incorporate infrastructure components before publishing.
 
-## Site Network Service (SNS)
+## Site Network Service considerations
 
-It's recommended to have a single SNS for the entire site, including the infrastructure.
+It is recommended to have a single SNS for the entire AOSM site, including the infrastructure. The SNS should deploy any infrastructure required (e.g., NAKS/AKS clusters, virtual machines, etc.), and then deploy the network functions required on top. Such design guarantees consistent and repeatable deployment of the network service to a given site from a single SNS PUT.
```

Comment:
Run Acrolinx against this file to catch stuff like: It is s/b it's and tense considerations.